manifests/daemonset_openshift.yaml

```diff
@@ -172,3 +172,23 @@ spec:
       - name: cnibin
         hostPath:
           path: /opt/cni/bin
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: bridge-filtering
+allowPrivilegedContainer: true
+allowHostDirVolumePlugin: true
+readOnlyRootFilesystem: false
+allowHostIPC: false
+allowHostNetwork: true
+allowHostPID: false
+allowHostPorts: false
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+users:
+- system:serviceaccount:bridge-filtering-cni:bridge-filtering
+volumes:
+- "*"
```

Comment:
Please allow only the host path type - afaiu, you don't use the others.

Reply:
Yeah, this is odd. I'm unable to make it work with only `hostPath` specified.

Reply:
Indeed. I'm gettting

```
Error creating: pods "bridge-filtering-ds-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used, spec.volumes[1]: Invalid value: "hostPath": hostPath volumes are not allowed to be used, spec.initContainers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed, provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "kubevirt-controller": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, spec.volumes[2]: Invalid value: "projected": projected volumes are not allowed to be used, provider "kubevirt-handler": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount]
```
for

```
apiVersion: security.openshift.io/v1
kind: SecurityContextConstraints
metadata:
  name: bridge-filtering
allowPrivilegedContainer: true
allowHostDirVolumePlugin: true
readOnlyRootFilesystem: false
allowHostIPC: false
allowHostNetwork: false
allowHostPID: false
allowHostPorts: false
runAsUser:
  type: RunAsAny
seLinuxContext:
  type: RunAsAny
users:
  - system:serviceaccount:bridge-filtering-cni:bridge-filtering
volumes:
  - hostPath
```

Which, as far as I can see is indeed the same as [the one you use in macvtap](https://www.diffchecker.com/cYq1jPhA)
 (apart from different order of two fields and different name/namespace).

Reply:
Hmm, that looks like the issue I'm hitting. Although it claims to be fixed in 4.9, I'm using 4.10.
I'll try opening a new BZ for 4.10,

Reply:
Turns out, `projected` volume needs to be allowed as well.